blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/BlueOceanRootAction.java

```diff
@@ -28,7 +29,12 @@
 @Extension
 public class BlueOceanRootAction implements UnprotectedRootAction, StaplerProxy {
     private static final String URL_BASE="blue";
-    private static final Random randomBits = new Random();
+
+    // Used to indicate UI should send refresh
+    @SuppressFBWarnings(
+            value="DMI_RANDOM_USED_ONLY_ONCE",
+            justification="random long value is used as salt, generated once in Jenkins instance")
+    private static final long randomBits = new Random().nextLong();
```

Comment:
Idea behind this is that randomBits is supposed to be a random constant - unique across Jenkins restart. There is sha1 hash calculated down below and set as value of X-Blueocean-Refresher header. Browser does a refresh anytime this value is different - thinking Jenkins restarted. 

The change I made to compute random long for every request was tripping UI to send page refresh on every request, ATH caught it in one of the test.